packs/windows-iad-sami-compliance.conf

```diff
@@ -0,0 +1,218 @@
+// inspired from https://github.com/iadgov/Splunk-Assessment-of-Mitigation-Implementations/raw/master/SAMI_Business_Logic.xlsx
+// default interval 1d=86400sec
+{
+  "platform": "windows",
+  "queries": {
+    // AE
+    "win_compliance_secureboot": {
```

Comment:
Perhaps we could just leave the pack name as `win_compliance`, and drop the `win_compliance_` prefix on all of these, as it seems a bit redundant.

Reply:
fine to me